Makefile

```diff
@@ -552,6 +553,15 @@ docker-remove: ## Remove the docker image
 	fi
 
 
+## --------------------------------------
+## Clean and verify
+## --------------------------------------
+
+.PHONY: vulncheck
+vulncheck:
```

Comment:
Let's rename this target to `vulncheck-go` as we may introduce more vulnerability checking for other things later.